### DISCUSSION TITLE
How do I specify the submitCommitAggregate fee deduction address for ProveCommitSectors3

### DISCUSSION THREAD

**Discussion by sokach-dev** [filecoin-project/lotus#12458] [General Q&A] [answered]
How do I specify the submitCommitAggregate fee deduction address for ProveCommitSectors3, which used to be control-1, but now it is deducted from the miner's available balance.

lotus-miner version 1.28.1
boostd version 2.3.0
lotus version v1.28.2

![image](https://github.com/user-attachments/assets/36834e53-87de-4c80-8662-338bea2dd335)

**Comment by beck-8** [comment, marked as answer]
update to [v1.28.2](https://github.com/filecoin-project/lotus/releases/tag/v1.28.2)

**Reply by sokach-dev** [reply]
Is it related to this PR? #12341

**Reply by beck-8** [reply]
yes
